Makefile

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+MK_DIR :=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+DISTRO ?= centos
+DISTRO_ROOTFS := "$(PWD)/$(DISTRO)_rootfs"
+IMG_SIZE=500
```

Comment:
nit, we can use resize2fs to resize the image to a minimal size for better shipping experience. This can be done in later PR though.